assets/sass/_variables.scss

```diff
@@ -7,6 +7,9 @@ $color-primary: #087acc;
 $color-primary-dark: #0769b0;
 $color-primary-light: #b3e5fc;
 
+$color-link-blue: #0b70b9;
```

Comment:
variable names shoud not have the colour in them (i.e. blue or grey) Please can you make them agnostic